techniques/general/G217.html

```diff
@@ -2,137 +2,114 @@
-               Where reliable information from the authors is not available, the presence of shortcuts
-               can possibly be identified by checking code (for example, in javascript, the presence
-               of <code>keydown</code>, <code>keyup</code> and <code>keypress</code> listeners). Another means of identifying shortcuts is to review documentation. Where
-               none of these strategies provide information on the presence of keyboard shortcuts,
-               manual tests will need to be completed to search for the existence of character key
-               shortcuts. Review the test procedure in <a href="https://w3c.github.io/wcag/techniques/failures/F99">F99: Failure of Success Criterion 2.1.4 due to implementing character key shortcuts that
-                  cannot be turned off or remapped</a> for guidance.
+            <p>テスタに​​とっての最初の課題は、コンテンツ制作者が作成したショートカットが存在するかどうかを特定し、既存のショートカットが修飾子なしの文字キーによってトリガされるかどうかを判断することである。
+               テスタが開発チームにアクセスできる場合、ショートカットキーの存在と性質は、多くの場合、関係するデザイナや開発者に尋ねることで判断できる。
+               コンテンツ制作者からの信頼できる情報が利用できない場合、ショートカットの存在はコードをチェックすることでおそらく識別できる (たとえば、javascriptで、<code>keydown</code>、<code>keyup</code>、および <code>keypress</code> listenerの存在)。
+               ショートカットを識別するもう 1 つの方法は、文書を確認することです。
+               これらの方法のいずれもキーボードショートカットの存在に関する情報を提供しない場合、文字キーショートカットの存在を探すために手動テストを完了する必要がある。
+               テスト手順については、<a href="../failures/F99">F99: 達成基準 2.1.4 の失敗例 － オフまたは再マップできない文字キーショートカットを実装している</a>のガイダンスを確認。
             </p>
          </section>
          <section id="examples">
-            <h2>Examples</h2>
+            <h2>事例</h2>
             <section class="example" id="example-1-author-provides-a-simple-mechanism-to-disable-shortcuts-in-a-web-based-email-client">
-               <h3>Example 1: Author provides a simple mechanism to disable shortcuts in a web-based
-                  email client
+               <h3>事例 1: コンテンツ制作者は、ウェブベースの電子メールクライアントでショートカットを無効にする簡単なメカニズムを提供する
                </h3>
-               <p>An email client contains single-character shortcuts so that keyboard users can more
-                  rapidly interact with the application (e.g., the R key replies to the current item
-                  in the inbox, the D key deletes it, the S key stars it). These keys do not apply while
-                  users are composing emails; however, users may become confused about whether the keyboard
-                  focus is in a compose pane or the inbox and, while in the inbox, inadvertently delete
-                  a message by trying to compose a word containing the letter D. To overcome this problem,
-                  the author provides a toggle button for the shortcuts, allowing users to easily turn
-                  them off or on.
+               <p>電子メールクライアントには 1 文字のショートカットが含まれているため、キーボード利用者はより迅速にアプリケーションと対話できる。
+                   (たとえば、R キーは受信トレイ内の現在のアイテムに応答し、D キーはそれを削除し、S キーはそれにスターを付ける)。
```

Comment:
ここではメールの話なので、replyは「返信」が適当でしょう。

```suggestion
                   (例えば、R キーは受信トレイ内の現在のアイテムに返信し、D キーは削除し、S キーはスターを付ける)。
```

Reply:
対応しました。ご確認よろしくおねがいします。
→　https://github.com/waic/wcag21/pull/351/commits/068389f98839bb9f289a1520975aa528f9ed655b